src/tests/integrity.test.ts

```diff
@@ -41,6 +36,27 @@ describe('Normal flow', (): void => {
             .to.be.instanceOf(SequelizeUser)
             .and.to.have.property('id')
             .that.is.a('number')
-            .that.is.equal(secondUser.id);
+            .and.is.equal(secondUser.id);
+    });
+
+    it('should be able to create a user, issue a token for it and' +
+        ' authenticate the token', async (): Promise<void> => {
```

Comment:
looks like extra space in the beginning of the string

Reply:
@hesamd 
It's appending to the string in previous line, so we need it.